### ISSUE TITLE
Option for Fix source

### ISSUE THREAD

**Issue by heyshaquib** [KotatsuApp/Kotatsu#1175]
### Describe your suggested feature

What if we can we option to select specific source to use when using 'Fix' Source option in the 3 dot menu? It would be pretty good having an option to select a specific source instead of it being randomly selected(which is how it is as of now afaik?)

### Acknowledgements

- [X] This is not a duplicate of an existing issue. Please look through the list of [open issues](https://github.com/KotatsuApp/Kotatsu/issues) before creating a new one.

**Comment by kuflierl** [comment]
That's what the "alternatives" feature does tho?

**Comment by heyshaquib** [comment]
> That's what the "alternatives" feature does tho?

You are correct. But i was talking about choosing which source to use as an alternative. What you mean when you are talking about 'alternative' is the option to manually selecting different source for every single thing you read..but i was talking about the 'Fix' option in home page.. although it does change source of all the broken stuff automatically, but that's the thing. It changes the source to whatever is the first one that works and not let us choose which source we say it should use. Let's say i have 10 manhua i was reading in the past on mangaowl, but that website is dead afaik. So i would like to automatically change the source of all the 10 thing to a source like LikeManga or something else which is fast and reliable. Does that make sense? Idk. Would it be nice to have feature? Sure. Is the app still perfect? Ofcourse.
